samples/generated/conftest.py

```diff
@@ -0,0 +1,293 @@
+# Copyright 2024 Google LLC All rights reserved.
```

Comment:
Should this be moved to https://github.com/googleapis/python-spanner/tree/main/samples instead of https://github.com/googleapis/python-spanner/tree/main/samples/generated since the actual samples are handwritten?

Reply:
+1
My concern is we have duplicated files like conftest, noxfile etc. This would bring in burden of maintaining files at 2 places.
Can we move this to samples/samples/admin/generated? and share the same nox and conftest file that are there in samples/samples?

Reply:
Moved to samples/samples/admin/generated